src/service/service.py

```
@@ -200,6 +201,32 @@ async def _handle_input(
+        credit_service = get_credit_service()
+        balance = await credit_service.get_balance(user_id, token=auth_token)
+        input["credit_state"] = {
+            "balance": balance.total_available,
+            "total_cost": 0,
+            "cost_per_node": {},
+            "difficulty": "easy",
+            "insufficient": False,
+            "stopped_at_feature": None,
+        }
+        logger.info(f"_handle_input: credit_balance={balance.total_available}")
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)